build.sbt

```diff
@@ -19,17 +15,15 @@ lazy val root = project
   .settings(name := "pekko-persistence-cassandra-root", publish / skip := true)
 
 lazy val dumpSchema = taskKey[Unit]("Dumps cassandra schema for docs")
-dumpSchema := (core / Test / runMain).toTask(" akka.persistence.cassandra.PrintCreateStatements").value
+dumpSchema := (core / Test / runMain).toTask(" org.apache.pekko.persistence.cassandra.PrintCreateStatements").value
 
 lazy val core = project
   .in(file("core"))
   .enablePlugins(Common, AutomateHeaderPlugin, MultiJvmPlugin)
   .dependsOn(cassandraLauncher % Test)
   .settings(
     name := "pekko-persistence-cassandra",
-    libraryDependencies ++= Dependencies.akkaPersistenceCassandraDependencies,
-    Compile / packageBin / packageOptions += Package.ManifestAttributes(
```

Comment:
Why was this removed, or asked differently what is `"Automatic-Module-Name"` for?

Reply:
pekko.persistence.cassandra would be a good value

Reply:
I will add that back. If "Automatic-Module-Name" is in the manifest then the module name is taken from it, otherwise the module name is derived from the JAR file name.